force-app/main/default/classes/ListDonation_TEST.cls

```diff
@@ -0,0 +1,53 @@
+@IsTest
+public class ListDonation_TEST {
+
+    @IsTest
+    public static void testGetDonationHistory() {
+        Test.startTest();
+		
+		Date myDateTime = Date.newInstance(2020, 2, 2);
+
+        Account mockDefaultAccount = new Account(Name = 'Default Account');
+        insert mockDefaultAccount;
+
+		Contact mockDefaultContact = new Contact(FirstName='Alice', LastName='Patterson', AccountId=mockDefaultAccount.Id);
+		Contact secondaryContact = new Contact(FirstName='Josh', LastName='Patterson', AccountId=mockDefaultAccount.Id);
+		insert mockDefaultContact;
+		insert secondaryContact;
+
+		Opportunity opportunityHardCredit = new Opportunity(Name='HardCredit1', AccountId=mockDefaultAccount.Id, Primary_Contact__c=mockDefaultContact.Id, Amount=400, StageName='Closed Won', CloseDate=myDateTime);
```

Comment:
```suggestion
		Opportunity opportunityHardCredit = new Opportunity(
		    Name='HardCredit1', 
		    AccountId=mockDefaultAccount.Id, 
		    Primary_Contact__c=mockDefaultContact.Id, 
		    Amount=400, 
		    StageName='Closed Won', 
		    CloseDate=myDateTime
		);
```

Suggestion for readability on these opportunities that are being constructed here.